releasenotes/notes/protein-folding-result-b344ac3c7f48e3ca.yaml

```diff
@@ -0,0 +1,34 @@
+---
+features:
+  - |
```

Comment:
I was looking through the release note one last time. One thing I guess that stands out is that we jump right into talking about a ProteinFaoldingResult as the first sentence- do you think you could add a sentence of two that presents an overview of the feature that is added - I am thinking something like.

"Added function to interpret the raw result of solving a protein folding problem such that the a solution is provided at the same level as the problem definition. Data for the folded protein is provided in x,y,z coordinate form and a figure can be obtained allowing a 3D plot to be rendered."


So it describes the overall feature and then the detail follows.